README.md

```diff
@@ -32,6 +32,12 @@ That's why I made this fork:
 
 ## How to use it
 
+### Requirements
+
+By default, this lib will use the [Tidy extension](https://github.com/htacg/tidy-html5) if it's available. Tidy is only used to cleanup the given HTML and avoid problems with bad HTML structure, etc .. It'll be suggested by Composer.
+
```

Comment:
Should not it be added  in `composer.json`  like `php-readability`  ?

```
"suggest": {
        "ext-tidy": "Used to clean up given HTML and to avoid problems with bad HTML structure."
}
```

Reply:
Since graby require php-readability, the message already exists when you install graby:

```
j0k3r/php-readability suggests installing ext-tidy (Used to clean up given HTML and to avoid problems with bad HTML structure.)
```

Reply:
ok